libse/SubtitleFormats/NetflixTimedText.cs

```diff
@@ -329,13 +329,7 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
             subtitle.Paragraphs.ForEach(p => p.Text = Regex.Replace(p.Text, @"^({\\an[1-7,9]})", string.Empty));
         }
 
-        public override bool HasStyleSupport
-        {
-            get
-            {
-                return false;
-            }
-        }
+        public override bool HasStyleSupport => false;
```

Comment:
I think this one can be removed too as SubtitleFormat `HasStyleSupport` is also default false

Reply:
**NetflixTimedText** inherits from **TimedText10**,
```
public class NetflixTimedText : TimedText10
```
**TimedText10** overrides `HasStyleSupport`,
```
public class TimedText10 : SubtitleFormat

public override bool HasStyleSupport => Configuration.Settings.SubtitleSettings.TimedText10ShowStyleAndLanguage;
```